public/static/docs/api-reference/open.md

```diff
@@ -0,0 +1,178 @@
+# dvc.api.open()
+
+[Context manager](https://www.python.org/dev/peps/pep-0343/#context-managers-in-the-standard-library)
+to open a tracked file.
+
+## Usage
+
+```py
+import dvc.api
+
+with dvc.api.open(
+        'get-started/data.xml',
+        repo='https://github.com/iterative/dataset-registry'
+        ) as fd:
+    # ... Process data
+```
+
+### Signature
+
+```py
+open(path, repo=None, rev=None, remote=None, mode="r", encoding=None)
+```
+
+### Types
+
+All **parameter** types are
+[string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str).
+
+The **return** type is a
+[file object](https://docs.python.org/3/glossary.html#term-file-object).
+
+No exceptions are raised by this function.
+
+## Description
+
+This function is analogous to the
+[`open()`](https://docs.python.org/3/library/functions.html#open) Python
+builtin, but for files tracked in <abbr>DVC projects</abbr> (by DVC or Git).
+However, it may only be used as a _context manager_ (using the `with` keyword as
+shown in the [Examples](#examples) below). There is no `dvc.api.close()`.
+
+> See also `dvc.api.read()` for a shorthand way to read the complete contents of
```

Comment:
no need to put a note, simplify:

`Use read to get the file content as a sting`

Reply:
I simplified it but the block quote seems relevant? We use them for notes that are related or important however not the main purpose of the section, in this case we're giving a open() example so read() is out of scope – just worth mentioning and linking to from here.

Reply:
👍